agenda.html

```diff
@@ -1,5 +1,163 @@
 ---
 permalink: /agenda
+layout: default
+schedule:
+- title: "Day 1: Presentation Day 1"
+- agenda:
+  - time: "08:15–08:45AM"
```

Comment:
Shouldn't each day be a single YAML object with properties `title` and `agenda`? Meaning, no `-` in front of agenda?

Reply:
Pushed the fix myself after giving it a run-through.

It didn't look wrong in the displayed page, but yeah, you were getting one item with a title and then an empty agenda list, followed by an empty heading and the agenda.

Reply:
Errr… trying to push. If I can figure out git.

Reply:
Yep, you are correct, I think I accidentally appended a hyphen in there when copy and pasting the days. Good catch!